### PR TITLE
DROOLS-1460 Additional test, combining multiple DMN models

### PR DESCRIPTION
**Pull request by tarilabs**

### CODE REVIEW

**Comment by tarilabs** [comment]
@etirelli can you kindly consider for approval+merge, please? With this test I ensure when more than a single .dmn model is in the kjar, it is resolved correctly by API.

(there are other tests I'm planning to add, later)

File: kie-server-parent/kie-server-tests/kie-server-integ-tests-dmn/src/test/java/org/kie/server/integrationtests/dmn/DMNTwoDMNModelsIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    }
+
+    @Test
+    public void test_evaluateAllDecisionsOnInputDataStringModel() {
+        DMNContext dmnContext = dmnClient.newContext();
+        dmnContext.set( "Full Name", "John Doe" );
+        ServiceResponse<DMNResult> evaluateAllDecisions = dmnClient.evaluateAllDecisions(CONTAINER_1_ID, 
+                "https://github.com/kiegroup/kie-dmn/input-data-string", "input-data-string",
+                dmnContext);
+
```
Comment (etirelli) [inline kie-server-parent/kie-server-tests/kie-server-integ-tests-dmn/src/test/java/org/kie/server/integrationtests/dmn/DMNTwoDMNModelsIntegrationTest.java]: @tarilabs I usually like to check that the ServiceResponse succeeded before doing any other assertions. It helps to quickly narrow down environment issues when the test suddenly starts failing for no apparent reason.

File: kie-server-parent/kie-server-tests/kie-server-integ-tests-dmn/src/test/java/org/kie/server/integrationtests/dmn/DMNTwoDMNModelsIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    
+    @Test
+    public void test_evaluateAllDecisionsOnFunctionDefinitionModel() {
+        DMNContext dmnContext = dmnClient.newContext();
+        dmnContext.set( "a", 10 );
+        dmnContext.set( "b", 5 );
+        ServiceResponse<DMNResult> evaluateAllDecisions = dmnClient.evaluateAllDecisions(CONTAINER_1_ID,
+                "https://www.drools.org/kie-dmn/function-definition", "function-definition",
+                dmnContext);
+
```
Comment (etirelli) [inline kie-server-parent/kie-server-tests/kie-server-integ-tests-dmn/src/test/java/org/kie/server/integrationtests/dmn/DMNTwoDMNModelsIntegrationTest.java]: same here.

**Comment by etirelli** [comment]
Looks good. +1.

**Comment by tarilabs** [comment]
@etirelli Added check that the ServiceResponse succeeded before doing any other assertions with commit ce8f3c9 .

**Pull request merged**